apps/vmq_swc/src/vmq_swc_proxy_diameter_transport.erl

```diff
@@ -0,0 +1,171 @@
+%% Copyright 2018 Octavo Labs AG Zurich Switzerland (https://octavolabs.com)
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(vmq_swc_proxy_diameter_transport).
```

Comment:
why the `proxy` part of the name? Is there a proxy which can then use different transports?

Reply:
the vmq_swc_proxy_diameter_transport is only used for testing to simplify network partitions. Eventually this module should go to the tests folder